apps/studio/prisma/schema.prisma

```diff
@@ -33,6 +33,16 @@ model VerificationToken {
   expires    DateTime
 }
 
+model Version {
+  id          BigInt     @id @default(autoincrement())
+  versionNum  BigInt
```

Comment:
question: why do we need BigInt for this? I suspect that there shouldn't be too many versions for a particular resource, and we will need to perform arithmetic operations on this field.

Reply:
Good point, what should be BigInt is the global id, not the resource-specific one. Will update!